lang/test/org/partiql/lang/eval/LikePredicateTest.kt

```diff
@@ -19,6 +19,7 @@ import org.partiql.lang.errors.*
 import org.partiql.lang.util.*
 import org.assertj.core.api.*
 import org.junit.*
+import kotlin.concurrent.thread
```

Comment:
Why is this imported? Could be a leftover from the other [PR](https://github.com/partiql/partiql-lang-kotlin/pull/279/files#diff-0a225530a11dc2cdc828cc9ec40aaab4)?

Reply:
yes, thanks.  will remove